File: python/Logic-1/alarm_clock.py
def alarm_clock(day, vacation):
    if vacation:
        if day > 0 and day < 6:
            return "10:00"
        else:
            return "off"
    else:
        if day > 0 and day < 6:
            return "7:00"
        else:
            return "10:00"

File: java/Logic-1/alarmClock.java
public String alarmClock(int day, boolean vacation) {
    if(vacation) {
        return (day >= 1 && day <= 5)? "10:00" : "off";
    } 
    return (day >= 1 && day <= 5)? "7:00" : "10:00";
}
